st2client/st2client/base.py

```diff
@@ -23,6 +23,9 @@
 
 import six
 import requests
+import urllib3
```

Comment:
@Kami If we revert the requests back to previous version in https://github.com/StackStorm/st2/pull/3597, this part probably should be reverted as well.